controllers/topics.controllers.js

```diff
@@ -9,4 +10,11 @@ exports.getTopics = (req, res, next) => {
     .catch((err) => {
         next(err);
     })
+}
+
+// this controller should be in it's own file?
```

Comment:
Don't leave comments in that don't mean anything. Yes controllers that have different jobs should be in different files. Yes you'll need to update it every time. And Yes you should really test that the new endpoint in the json file is there.